StackOnUList/MyStack.cpp

```diff
@@ -0,0 +1,98 @@
+#include "StdAfx.h"
+#include "MyStack.h"
+
+
+MyStack::MyStack(void): s_size(0), s_top(NULL)
```

Comment:
Personaly I prefer writing initialization list in a row and formt it like this
: s_size(0)
, s_top(NULL)
works for huge lists easy to delete or add somehing